packages/labs/react/src/create-component.ts

```diff
@@ -174,7 +178,7 @@ export const createComponent = <I extends HTMLElement, E>(
         setProperty(
           this._element,
           prop,
-          this.props[prop as keyof ComponentProps],
+          this.props[prop as Exclude<keyof ComponentProps, symbol>],
```

Comment:
What is this doing?

Reply:
I was getting an error that symbol couldn't be used in an index type. I scanned, but couldn't figure which part of React.HTMLAttributes<HTMLElement> was including a symbol key, so I just... excluded it

Reply:
I removed it and don't see the error now 🤷